DataFormats/PatCandidates/interface/VIDCutFlowResult.h

```diff
@@ -0,0 +1,60 @@
+#ifndef __DataFormats_PatCandidates_VIDResult_H__
+#define __DataFormats_PatCandidates_VIDResult_H__
+
+#include <unordered_map>
```

Comment:
On 6/25/15 2:52 AM, Lindsey Gray wrote:
> In DataFormats/PatCandidates/interface/VIDCutFlowResult.h
> <https://github.com/cms-sw/cmssw/pull/9659#discussion_r33229879>:
> 
>> @@ -0,0 +1,60 @@
>> +#ifndef __DataFormats_PatCandidates_VIDResult_H__
>> +#define __DataFormats_PatCandidates_VIDResult_H__
>> +
>> +#include <unordered_map>
> 
> Ah ok, I can just move to std::map instead of unordered_map. Then revert
> later with an i/o rule.
> 


will the io rule actually work?
The main point is to be able to read objects produced in root6 with root5
(the other way around is less interesting)


> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/9659/files#r33229879>.
>